common-gradle-enterprise-maven-configuration/.mvn/gradle-enterprise.xml

```diff
@@ -32,16 +32,19 @@
       <enabled>true</enabled>
     </local>
     <remote>
+     <!-- Use the Gradle Enterprise connector's access key based authentication -->
+      <enabled>true</enabled>
+      <storeEnabled>#{isTrue(env['CI'])}</storeEnabled> <!-- adjust to your CI provider -->
+
+      <!-- Use Gradle's built-in HTTP connector
```

Comment:
in this case it isn't "Gradle's built-in HTTP connector" let's name it something else. "Use build cache node credentials"?